.env.vagrant

```diff
@@ -0,0 +1,16 @@
+DEBUG=true
+SECRET_KEY='dev'
+DATABASE_URL='sqlite:////home/vagrant/opencraft.db'
```

Comment:
@brousch We should not use sqlite - there are too many differences between it and a proper postgresql DB. If people use it to develop, they will miss issues (like enforcing character limits on fields) because sqlite gnores them, while a proper DB doesn't.